core/extrinsics_submission_service/extrinsic_submission_api.hpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_EXTRINSIC_SUBMISSION_API_HPP
```

Comment:
```suggestion
#ifndef KAGOME_EXTRINSIC_SUBMISSION_API_HPP
```

Reply:
Too long :(

Reply:
Doesn't matter. Even better if it is long (no collisions will occur)

Reply:
I guess it can be marked as resolved?